app/models/article.rb

```diff
@@ -1,39 +1,39 @@
 class Article < ActiveRecord::Base
-  searchkick autocomplete: ['title']
+  searchkick autocomplete: ["title"]
   extend FriendlyId
 
   has_many :article_categories
   has_many :categories, through: :article_categories
   has_many :comments, dependent: :destroy
-  # TODO questo è l'autore? Se sì ti consiglio di chiamare la relazione author altrimenti tra un po' è un casino
+  # TODO: questo e l'autore? Se sì ti consiglio di chiamare la relazione author altrimenti tra un po" è un casino
   belongs_to :user
 
   friendly_id :title, use: [:slugged, :finders]
 
   has_attached_file :image, styles: {
-    thumb: '200x200>',
-    medium: '500x500#',
-    large: '1000x1000>'
+    thumb: "200x200>",
+    medium: "500x500#",
+    large: "1000x1000>"
   }, :path => "blogails/:class/:attachment/:id/:style/:filename.:extension", :url => ":s3_domain_url"
 
   # Validate the attached image is image/jpg, image/png, etc
   validates_attachment_content_type :image, :content_type => /\Aimage\/.*\Z/
 
-  # TODO This should all be scopes
-########### ARTICOLI VISIBILI SOLO CON ORARIO E DATA MINORE/UGUALE
+  # TODO: This should all be scopes
+  ########### ARTICOLI VISIBILI SOLO CON ORARIO E DATA MINORE/UGUALE
   def self.available
-    where('date <= ?', Time.now)
+    where("date <= ?", Time.now)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.